core/services/keystore/eth.go

```diff
@@ -109,6 +109,7 @@ func (ks *eth) Create(chainIDs ...*big.Int) (ethkey.KeyV2, error) {
 	if err == nil {
 		ks.notify()
 	}
+	ks.logger.Infow(fmt.Sprintf("Created EVM key with ID %s", key.Address.Hex()), "address", key.Address.Hex(), "evmChainIDs", chainIDs)
```

Comment:
What format does `chainIDs` get logged in? JSON array?
Might we later get annoyed by the plural key name `evmChainIDs` when trying to filter logs by keys and `evmChainID` doesn't match this?

Reply:
mm, good point

Reply:
Yes, JSON array.

I think the plural is ok.